src/app/features/exchange-set/ess-list-encs/ess-list-encs.component.ts

```diff
@@ -30,6 +30,7 @@ export class EssListEncsComponent implements OnInit {
   selectedEncList: string[];
   displaySingleEncVal: boolean = false;
   public displaySelectedTableColumns = ['enc', 'X'];
+  estimatedSizeofENC: string="0KB";
```

Comment:
Please rename to `estimatedTotalSize`
```suggestion
  estimatedTotalSize: string="0KB";
```

Reply:
resolved